src/app/context/PatronContext.jsx

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import React, { createContext } from 'react';
+
+/**
+ * Wrapper context component to get patron data from the Redux store.
```

Comment:
it's probably out of scope for this ticket, but would it be a lot of work to refactor all patron logic out of redux into the context?